paprika/tleap.py

```diff
@@ -184,12 +184,19 @@ def filter_template(self):
             # Find loadpdb line, replace pdb file if necessary, set unit name
             if re.search("loadpdb", line):
                 words = line.rstrip().replace("=", " ").split()
-                if self.loadpdb_file is None:
-                    self.loadpdb_file = words[2]
                 self.unit = words[0]
                 filtered_lines.append(
-                    "{} = loadpdb {}".format(self.unit, self.loadpdb_file)
+                    "{} = loadpdb {}\n".format(
+                        self.unit, self.loadpdb_file if self.loadpdb_file else words[2]
```

Comment:
If there are multiple `loadpdb` commands would all these be replaced by `self.loadpdb_file` if specified?

Reply:
If `self.loadpdb` is specified, yes. I figured that if you specify `self.loadpdb` then you want that to be the one that `tleap` will use, discarding whatever is in the template. If you don't specify `self.loadpdb`, then whatever is in the template will be carried over. I guess it would be a really edge case to specify `self.loadpdb` and multiple `loadpdb` lines in tleap. Do you have an alternative in mind?

Reply:
Would it be better to ignore `self.loadpdb_file` if there are multiple `loadpdb` commands in the `template_lines`?

Reply:
What do you think about it now? Too convoluted or okay?

Reply:
Looks good!